app/Models/Wiki/Redirect.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ *    Copyright 2015-2017 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace App\Models\Wiki;
+
+use App\Libraries\OsuWiki;
+use Cache;
+
+class Redirect extends Page
```

Comment:
~~I have not a clue (it just works for me with it there ¯\\\_(ツ)\_\/¯)~~
I guess it is to use one of the cache functions (`cacheKeyPage()`) from Page, otherwise I will need to define it or something?

Also, I left an `or true` expression on line 39... whoops